### DISCUSSION TITLE
TagInput component

### DISCUSSION THREAD

**Discussion by oxorp** [mantinedev/mantine#1056] [Feature requests]
We're currently evaluating mantine for rewriting an enterprise logistics management software and it's a blast. We found everything we need for our forms so far except a Tag-Input. The current version of our application uses [react-taginsput](https://www.npmjs.com/package/react-tagsinput) and we're loving it for stuff like `validationRegex` and `pasteSplit`.

Style wise it would look much like `MultiSelect` with `creatable` I guess.

Let me know if you'd be interested in adding this to mantine and we'll have a developer working on a PR asap. 

Cheers!

**Comment by rtivital** [comment]
Sounds fine, feel free to submit a PR and reach out on Discord if you need help in contributing

**Comment by auronsan** [comment]
https://github.com/mantinedev/mantine/pull/877

**Reply by rcronin** [reply]
Did this ever make it into the code base?

**Comment by paulm17** [comment]
Would love to see the ability to set the field position, much like: https://github.com/react-tags/react-tags

For now, I'm using both TagInput and react-tags, although I'd much rather use Mantine Components.  😄

**Comment by affanshahid** [comment]
Is this available now? I saw the PR was indeed merged

**Comment by auronsan1st** [comment]
@paulm17  like this ? 
<img width="1227" alt="Screen Shot 2022-10-31 at 22 38 11" src="https://user-images.githubusercontent.com/60646861/199048223-16760e05-61c9-47b2-86a5-df1ec2b7a48d.png">

**Reply by paulm17** [reply]
Yeah like that.  

My use case is (see images).

The problem that I have, is that I'm allocating a confined space for the tags.  So the ability to also scroll the tags must be present also.

![bar](https://user-images.githubusercontent.com/387463/199049830-1c83e9b2-5f33-4bf6-8e8a-b92ec8d367ea.png)

![foo](https://cdn.discordapp.com/attachments/890965830727241785/1025481153135775835/Screenshot_2022-09-30_at_19.55.42.png)

**Reply by auronsan1st** [reply]
okay I added it
<img width="943" alt="Screen Shot 2022-11-11 at 22 37 31" src="https://user-images.githubusercontent.com/60646861/201374074-ccc31113-4751-48a5-b6b2-edda4a81734a.png">

**Reply by paulm17** [reply]
That's awesome, thanks so much!

**Comment by pbhopalka** [comment]
Any idea when this is going to official Mantine docs? I see in the PR, it's been added to Mantine Labs, but can't find a reference for it.

**Reply by rtivital** [reply]
Available in v7 – https://v7.mantine.dev/core/tags-input
